ui/src/stores/nodeStructureStore.ts

```diff
@@ -101,12 +112,12 @@ export const useNodeStructureStore = defineStore('nodeStructureStore', () => {
     }
   }
 
-  const setSelectedCategories = async (cats: Category[]) => {
-    queryFilter.value = {
-      ...queryFilter.value,
-      selectedCategories: [...cats]
-    }
-  }
+  // const setSelectedCategories = async (cats: Category[]) => {
```

Comment:
Should these commented-out lines be removed, or are they going to be implemented?